src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithplaintextcommentfilter/InputSuppressWithPlainTextCommentFilterCustomMessageFormat.java

```diff
@@ -1,15 +1,41 @@
+/*
+SuppressWithPlainTextCommentFilter
+offCommentFormat = // CHECKSTYLE:OFF
+onCommentFormat = // CHECKSTYLE:ON
+checkFormat = (default).*
+messageFormat = .*tab.*
+idFormat = (default)(null)
+
+
+com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck
+id = ignore
+format = .*[a-zA-Z][0-9].*
+message = (default)
+ignoreCase = (default)false
+minimum = (default)0
+maximum = (default)0
+fileExtensions = (default)all files
+
+
+com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck
+eachLine = true
+fileExtensions = (default)
+
+
+*/
+
 package com.puppycrawl.tools.checkstyle.filters.suppresswithplaintextcommentfilter;
 
 public class InputSuppressWithPlainTextCommentFilterCustomMessageFormat {
 
     // CHECKSTYLE:OFF
-    private int A1;
-
-	private static final int a1 = 5; // contains tab character
+    private int A1; // violation
 
-    int a2 = 100;
+	private static final int a1 = 5; // contains tab character // filtered violation
+    // violation above
```

Comment:
"violation" or "filtered violation" ? 

"filtered violation" means no violation when all config is used.

Reply:
we need to update our parser to treat `// filtered violation` as "// violation" if required.

Reply:
It has both actually. There are 2 violations on line 34, and only one gets filtered.

Reply:
There is no way I can add comment to cover both, so this is a workaround. I added one on the next line.

Reply:
Please put message to both lines to make it clear what is filtered and stay

Reply:
changed to 
```
	private static final int a1 = 5; // filtered violation 'contains a tab'
        // violation above 'illegal pattern'
```